client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -131,12 +137,14 @@ class RequiredPluginsInstallView extends Component {
 					progress,
 				} ) );
 				this.props.installPlugin( site.ID, wporgPlugin );
+				this.setUpdateTimeout();
 				return;
```

Comment:
Is there a reason why this is all serial instead of parallel? Granted, it would complicate the timeout situation, but if we could install plugins in parallel, it could greatly speed up the process.